kotlin-native/Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt

```diff
@@ -727,3 +735,43 @@ public fun COpaquePointer.readBytes(count: Int): ByteArray {
     nativeMemUtils.getByteArray(this.reinterpret<ByteVar>().pointed, result, count)
     return result
 }
+
+@ExperimentalForeignApi
+public fun setMemory(mem: NativePointed, value: Byte, size: Long) {
+    nativeMemUtils.memset(mem, value, size)
+}
+
+@ExperimentalForeignApi
+public fun copyMemory(destMem: NativePointed, srcMem: NativePointed, size: Long) {
+    nativeMemUtils.memcpy(destMem, srcMem, size)
+}
+
+@ExperimentalForeignApi
+public fun moveMemory(destMem: NativePointed, srcMem: NativePointed, size: Long) {
+    nativeMemUtils.memmove(destMem, srcMem, size)
+}
+
+@ExperimentalForeignApi
+public fun compareMemory(destMem: NativePointed, srcMem: NativePointed, size: Long): Int {
+    return nativeMemUtils.memcmp(destMem, srcMem, size)
+}
+
+@ExperimentalForeignApi
+public inline fun <reified T : CVariable> CPointer<T>.setBlock(value: Byte, length: Int) {
```

Comment:
Maybe just `fillWith`? "Block" is not really common term, and more a  straightforward `setMemoryBlock` starts to look verbose.

Reply:
Also a good name, will change.